docker/postgres/Dockerfile

```diff
@@ -0,0 +1,16 @@
+ARG POSTGRES_VERSION=16
+
+FROM postgis/postgis:${POSTGRES_VERSION}-3.4
+
+RUN apt-get update && apt-get install -y git build-essential software-properties-common postgresql-server-dev-${POSTGRES_VERSION} clang
+
+RUN git clone https://github.com/citusdata/pg_cron.git
+
+RUN cd pg_cron && make && make install
+
+RUN cd / && \
```

Comment:
Ideally, I aim to deploy this image seamlessly, requiring no manual tweaks to PostgreSQL configuration files. Upon installation, I would advisable to activate `pg_cron` within the PostgreSQL configuration by adjusting settings such as `shared_preload_libraries` and others as necessary (everything while building container).